jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -98,16 +119,20 @@ async def applications_delete_upload(
     """
     Delete application tarball using an authenticated user token.
     """
+    logger.debug(f"Preparing to delete tarball for {application_id=}")
+
     select_query = applications_table.select().where(applications_table.c.id == application_id)
     raw_application = await database.fetch_one(select_query)
+
     if not raw_application:
-        raise HTTPException(
-            status_code=status.HTTP_404_NOT_FOUND,
-            detail=f"Application {application_id=} not found.",
-        )
+        message = f"Application {application_id=} not found."
+        logger.warning(message)
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=message)
+
     application = ApplicationResponse.parse_obj(raw_application)
 
     if not application.application_uploaded:
+        logger.warning(f"Trying to delete a applications that was not uploaded ({application_id=})")
```

Comment:
The return code is 2XX, part of the "OK" class. It seems a bit too scary to send a warning as a log. I think `debug` is an appropriate level here.

Is this case (when entering the `if` block) an issue when using jobbergate?

Reply:
I agree, I'll change it to `debug`.

In this case, the `id` is provided by the users. If they want to delete something that was not uploaded or was already deleted, we can just stop the function.

Reply:
solved in [0c61e5d](https://github.com/omnivector-solutions/jobbergate/pull/121/commits/0c61e5d42e5c10b8193ec444d45138c631f1b87a)